src/ansys/fluent/core/services/datamodel_tui.py

```diff
@@ -197,6 +202,45 @@ def get_doc_string(self, include_unavailable: bool = False) -> str:
         response = self._service.get_attribute_value(request)
         return _convert_gvalue_to_value(response.value)
 
+    def get_static_info(self) -> Dict[str, Any]:
+        """Get static info at menu level.
+
+        Returns
+        -------
+        DataModelProtoModule.StaticInfo
+            static info
+        """
+        if hasattr(DataModelProtoModule, "GetStaticInfoRequest"):
```

Comment:
returns the info dictionary either using the new `GetStaticInfo` rpc (for 23.1) or recursively retrieving child item info (for 22.2)